tensorflow/compiler/jit/deadness_analysis.cc

```diff
@@ -250,7 +250,7 @@ class NotPredicate : public Predicate {
 class AndRecurrencePredicate : public Predicate {
  public:
   explicit AndRecurrencePredicate(int64 id, Predicate* start, Predicate* step,
-                                  std::vector<string> frame)
+                                  std::vector<string> &frame)
       : Predicate(id), operands_({start, step}), frame_(std::move(frame)) {}
```

Comment:
It was intentional to take `frame` by value.  That way if the caller passed a temporary then we don't copy anything.  I suspect the form you're suggesting will incur a copy.

Reply:
Agree.